src/ui/styles/_animations.css

```
@@ -93,6 +93,19 @@
     }
 }
 
+@keyframes updateDotFlash {
```

Stylelint (17.9.0)

[error] Expected keyframe name "updateDotFlash" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)